### ISSUE TITLE
ClobberWarnings vs protobuf

### ISSUE THREAD

**Issue by bollwyvl** [conda-forge/googleapis-common-protos-feedstock#50]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

It appears this package vendors a lot of content also included in the base `protobuf` package, which results in a huge number of `ClobberWarnings` ([log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=618957&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=15951)):

```
ClobberWarning: This transaction has incompatible packages due to a shared path.
  packages: conda-forge/linux-64::protobuf-3.20.2-py310hd8f1fbe_1, conda-forge/noarch::googleapis-common-protos-1.57.0-pyhd8ed1ab_1
  path: 'lib/python3.10/site-packages/google/protobuf/__init__.py'
```

and different at-rest versions of `protobuf` as found by `pip check` ([log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=618957&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=17004)):

```
+ pip check
dagster 1.1.3 has requirement protobuf<4,>=3.13.0, but you have protobuf 4.21.9.
```

### Installed packages

```shell
The following NEW packages will be INSTALLED:

    _libgcc_mutex:                      0.1-conda_forge             conda-forge
    _openmp_mutex:                      4.5-2_gnu                   conda-forge
    aiohttp:                            3.8.3-py310h5764c6d_1       conda-forge
    aiosignal:                          1.3.1-pyhd8ed1ab_0          conda-forge
    alembic:                            1.8.1-pyhd8ed1ab_0          conda-forge
    arrow-cpp:                          8.0.1-py310h4f1c134_2_cpu   conda-forge
    async-timeout:                      4.0.2-pyhd8ed1ab_0          conda-forge
    zipp:                               3.11.0-pyhd8ed1ab_0         conda-forge
    zlib:                               1.2.13-h166bdaf_4           conda-forge
    zstd:                               1.5.2-h6239696_4            conda-forge
```

### Environment info

```shell
show_channel_urls: True
conda_build:
  pkg_format: 2
  zstd_compression_level: 16

==> /home/conda/.condarc <==
pkgs_dirs:
  - /home/conda/feedstock_root/build_artifacts/pkg_cache
  - /opt/conda/pkgs
conda-build:
  root-dir: /home/conda/feedstock_root/build_artifacts

==> envvars <==
bld_path: /home/conda/feedstock_root/build_artifacts
```

**Comment by xylar** [comment]
@bollwyvl, I'm very sorry about this.  It seems to have resulted from #49, which switched to noarch.  I have a guess as to what might be going wrong. I can see the clobber warnings in our build logs, too:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=616609&view=logs&jobId=656edd35-690f-5c53-9ba3-09c10d0bea97&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d
I just didn't happen to notice that.  I'll make sure they're gone with the fix I have in mind.

**Comment by xylar** [comment]
@bollwyvl, let me know if #51 fixes the issue you're seeing once your CI starts picking it up.  I'll mark build 1 as broken as soon as I have a better internet connection (I'm on a train right now).

**Comment by bollwyvl** [comment]
ha, well, that dagster build takes ~1hr... no rush on my end!

**Comment by xylar** [comment]
I've requested that build 1 get marked as broken here:
https://github.com/conda-forge/admin-requests/pull/517

**Comment by xylar** [comment]
I'm going to close this for now.  @bollwyvl, let me know if it crops up again.

**Comment by xylar** [comment]
@ngam, thanks for reporting.  No, this doesn't sound benign and it seems to be the same problem again. :-(

**Comment by xylar** [comment]
I think what might have happened in #49 and #51 is that protobuf got installed from pip because it wasn't in the host environment.  I'm attempting to fix that in #52.  I will make sure things look okay in a local build.
